src/main/java/org/jabref/logic/formatter/bibtexfields/EscapeCurrencySymbolsFormatter.java

```diff
@@ -0,0 +1,37 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import java.util.Objects;
+import java.util.regex.Matcher;
+
+import org.jabref.logic.cleanup.Formatter;
+import org.jabref.logic.l10n.Localization;
+
+public class EscapeCurrencySymbolsFormatter extends Formatter {
```

Comment:
This should be called `EscapeDollagerSignFormatter`, because it "just" replaces the dolalr sign.